clients/banking/src/components/TransactionDetail.tsx

```diff
@@ -138,6 +141,10 @@ export const TransactionDetail = ({ transaction, large }: Props) => {
         <Tile
           style={styles.tile}
           footer={match(transaction)
+            // BankingFee should never happened and is a default value, so there isn't any wording for it
```

Comment:
@Epimodev I don't get the meaning of the comment here 😶

Reply:
yes I agree it's not clear. I just want to say this should not happen so we don't handle this case. It was approved by product team

Reply:
Ah, got it, `P.not("BankingFee")`

Reply:
ℹ️ just changed comment here: 1139da1a166d093e5983098578b4f9da81a671c9